src/home/NewsBanner.vue

```diff
@@ -135,8 +136,6 @@ export default class NewsBanner extends Vue {
 }
 
 .news-content {
-  height: 100%;
-  overflow-y: scroll;
-  scroll-behavior: smooth;
+  height: 98%;
```

Comment:
What does this do? I'm trying to see a difference but I can't.

Reply:
Could you show me what problem the 98% solves? I don't see the overflow. It's just css so it's not a big deal, but 2 years from now, someone's going to look at that code and not understand why the magic number 98 was chosen.

Reply:
Ok I think I see it. I think we can hide the x overflow too. It shows an ugly white box. Maybe style the scroll bar to be visible. But not on the settings page.

Reply:
We can fix that in another pr though. Nice job!